README.md

```diff
@@ -53,6 +53,69 @@ By default the `awsProxy` is set to `AWSProxy.APIGatewayV2`.
 - `AWSProxy.APIGatewayV2`
 - `AWSProxy.ALB`
 
+## Vite preset
+
+If you use Vite, then the `awsPreset` preset is an easy way to configure aws support.
+It will do a post remix build and create a handler function for use in aws lambda.
+
+There is no need for a separate `server.js` file. The preset will take care of that.
+However, if you want to manage your own `server.js` file, you can pas a custom `entryPoint` to your own `server.js`.
+
+```typescript
+import type { PluginOption } from 'vite'
+import type { Preset } from '@remix-run/dev'
+
+import { vitePlugin as remix } from '@remix-run/dev'
+import { awsPreset, AWSProxy } from 'remix-aws'
+import { defineConfig } from 'vite'
+
+export default defineConfig(
+    {
+        ...
+        plugins: [
+            remix({
+                presets: [
+                    awsPreset({
+                        // or AWSProxy.APIGatewayV2 or AWSProxy.ALB
+                        awsProxy: AWSProxy.APIGatewayV1,
+                        
+                        // additional esbuild configuration
+                        build: {
+                          minify: true,
+                          treeShaking: true,
+                          ...
+                        }
+                    }) as Preset
+                ]
+            }) as PluginOption,
+        ]
+    }
+)
+```
+
```

Comment:
Would be nice to have a section on local testing. What script targets would be appropriate to use?

Reply:
@meza good idea! 

I usually test this with sam cli so I can reproduce the same infrastructure as I would use in AWS.
I created a simple remix example project where I added some readme, predefined npm scripts and already configured the vite preset for testing. Please check the readme if you want to give it a spin.

👉 https://github.com/wingleung/remix-aws-example
👉 especially [this commit](https://github.com/wingleung/remix-aws-example/commit/6e38088152d6a5239507441bef6fd9d9b143c4ec) where I added the remix-aws setup

Reply:
thanks! Also is there a reason to stay with awsgateway v1?

Reply:
good point, I added a commit to make api gw v2 the default everywhere 👉 https://github.com/wingleung/remix-aws/pull/14/commits/e5fdefb15a1fce13112ae95f83fc26d42783c5d7

it was already the default in the vite preset and in the [createRequestHandler](https://github.com/wingleung/remix-aws/blob/main/src/server.ts#L48) and the [vite preset](https://github.com/wingleung/remix-aws/pull/14/files#diff-a8a8e7d4957999e2a30eb81a7886914b66875af7511b119e93fa46b2045cbc94R25)